uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-comments-tab.js

```diff
@@ -141,6 +141,7 @@ define([
     function loadCommentsAndContactNames () {
       return vm.request.loadComments()
         .then(function () {
+          $rootScope.$broadcast('Request Updated');
```

Comment:
namespace the events, i guess something like `LeaveRequestModal::dataChanged` or something like that (the idea being that it's an event that is related to the modal only, and it's triggered when something changed)